app.js

```diff
@@ -0,0 +1,80 @@
+const NOTES_KEY = "notes"
+function addTask(event) {
+    event.preventDefault();
+    const data = collectDataFromForm();
+    data.id = Date.now()
+    const newNote = generateNote(data);
+    
+    injectTRToDOM(newNote);
+    clearForm();
+    saveToLocalStorage(data);
+}
+function collectDataFromForm() {
+    const task = document.getElementById('task').value;
+    const date = document.getElementById('date').value;
+    const time = document.getElementById('time').value;
+    return {
+        task,
+        date,
+        time
+    }
+}
+function generateNote(data) {
+    
+    const {id, task, date, time } = data;
+
+    const newNote = `
+        <div class="note">
```

Comment:
you should have applied the `fadeIn` only on new notes. here for example when i simply refresh the page the existing notes load with animation